### DISCUSSION TITLE
Blog Posts

### DISCUSSION THREAD

**Discussion by reubenalfred** [Megabit/Blazorise#4840] [General]
Are the blog posts properly keyword researched before creating the post?

**Comment by stsrki** [comment]
Not really ;)

**Reply by reubenalfred** [reply]
I'd like to help the project Blazorise with this to bring in traffic. It is important to do kw research, build a hub and spokes model based blog posts.

If the blog posts are not built around kw that are being searched for then the posts won't get hits and raise brand awareness...

Let's talk about this, shall we?

It is also important to add analytics to track how the site is being used especially for the documentation and demo...

**Reply by stsrki** [reply]
We're open to any advice :)

> It is also important to add analytics to track how the site is being used especially for the documentation and demo...

For analytics, we use [Plausible.com](https://plausible.io/) which has some basic analytics capabilities.

**Reply by reubenalfred** [reply]
I have sent an email from the site's contact page. Please reply to it so that I can send my suggestions.
